openeo/rest/datacube.py

```diff
@@ -2477,6 +2477,7 @@ def execute_batch(
         job_options: Optional[dict] = None,
         validate: Optional[bool] = None,
         auto_add_save_result: bool = True,
+        log_error=True,
```

Comment:
I think `log_error` is not ideal as a name here, as we always log errors. This feature is just about automatically showing them or not. So I would name this `show_error_logs` instead.

Also, add a type hint, e.g. `show_error_logs: bool = True` for documentation purposes